test/test_methods_test.rb

```diff
@@ -11,6 +11,18 @@ def test_asserts_controller_submits_correct_form
     end
   end
 
+  def test_asserts_controller_submits_at_all
+    runner = FooControllerTest.new :foo
+
+    error = assert_raises MiniTest::Assertion do
+      runner.assert_submits FooForm do
+      end
+    end
+
+    assert_equal "Expected TestMethodsTest::FooController to submit a TestMethodsTest::FooForm",
+                 error.message
```

Comment:
The heredoc one has the weird heredoc syntax, but is it less cluttered in the end because the error message goes on the following line?